# Rnnotator
Rnnotator is an automated software pipeline that generates transcript models by de novo assembly of RNA-Seq data without the need for a reference genome.

Rnnotator must be run on a 64-bit Linux architecture. Before running Rnnotator the
following prerequisites must be installed:
• Blat v. 34 (http://genome.ucsc.edu/FAQ/FAQblat.html#blat3)
• Velvet 1.0.15 (http://www.ebi.ac.uk/~zerbino/velvet/)
• AMOS (http://sourceforge.net/apps/mediawiki/amos/index.php)
• Vmatch 2.0 (http://www.vmatch.de/)
• bwa 0.5.8c (http://bio-bwa.sourceforge.net/)
• MUMmer (http://sourceforge.net/projects/mummer/)
• BioPerl (http://www.bioperl.org)
• Perl modules: Parallel::ForkManager, Tree (http://search.cpan.org/)
Optional prerequisites are:
• Oases 0.1.18 (http://www.ebi.ac.uk/~zerbino/oases/)
• Bambus 2.33 (http://www.cbcb.umd.edu/software/bambus/)
• Sopra 1.0 (dayarian@physics.rutgers.edu) x1 – x4 scripts
